tests/test_external_search.py

```diff
@@ -924,7 +927,7 @@ def expect(availability, collection, works):
                 order=Facets.ORDER_TITLE
             )
             self._expect_results(
-                works, None, Filter(facets=facets), order=False
+                works, None, Filter(facets=facets), ordered=False
```

Comment:
This was always wrong but it didn't cause problems until I started giving `query_works` any leftover keyword arguments passed into `_expect_results`. "order" is a valid keyword argument for `query_works`, but that's not what's supposed to happen here.